src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaSelection/MediaSelection.js

```diff
@@ -26,8 +26,13 @@ export default class MediaSelection extends React.PureComponent<Props> {
             value,
             locale,
         } = this.props;
+        let selectedMediaIds = null;
 
-        this.mediaSelectionStore = new MediaSelectionStore(value, locale);
+        if (value && value.ids) {
+            selectedMediaIds = value.ids;
+        }
```

Comment:
How about `const selectedMediaIds = (value && value.ids) ? value.ids : null;`? Then it couldn't be changed by accident afterwards.